src/service.ts

```diff
@@ -0,0 +1,80 @@
+const urlRu: string = `https://travel-application-backend.herokuapp.com/countries/ru`;
```

Comment:
`: string ` is something that TS assumes automatically. You may drop that type definition.